lms/static/sass/discussion/_discussion.scss

```diff
@@ -241,20 +228,17 @@ body.discussion {
 
   .discussion-column {
     @include float(right);
-    @include box-sizing(border-box);
+    box-sizing: border-box;
+    padding-left: ($baseline/2);
```

Comment:
Any styles that affect only one side (right or left) should be RTL-ified, otherwise it'll look funky when viewing right-to-left. If you come across anything that isn't RTL-friendly we should either fix it up, or make a TODO or a ticket to get it fixed up.